sr_description/hand/xacro/finger/biotac/biotac.urdf.xacro

```diff
@@ -154,20 +154,20 @@ xmlns:geom="http://playerstage.sourceforge.net/gazebo/xmlschema/#geom">
         <inertia ixx="0.0" ixy="0.0" ixz="0.0" iyy="0.0" iyz="0.0"
         izz="0.0" />
       </inertial>
-      <visual>
+      <!--visual>
```

Comment:
do visual create collisions in moveit ? i doubt it. I guess since we use kdl IK for the biotac, the new chain to tip does not matter. On the other specific IK solver might have had problems with chain renaming in the tip joint, I am not sure. No problem here